.github/workflows/_charm-quality-checks.yaml

```diff
@@ -292,3 +292,24 @@ jobs:
         with:
           timeout-minutes: 30
           limit-access-to-actor: true
+
+      - name: Show logs
+        run: | 
+          if test -d ./.logs; then
+            for filename in ./.logs/*.txt; do
```

Comment:
Note that this is a now a required `.gitignore` addition to all of our charms; is there a way we can avoid that?

Reply:
could put all files in tmp, but it becomes hard to do that in a way which is both pytest-jubilant-agnostic and not easily conflicting with other things that might write to tmp

suggestions are welcome

Reply:
discussed and agreed to keep it here for the time being.